app/display/editor/src/main/java/org/csstudio/display/builder/editor/Preferences.java

```diff
@@ -20,9 +21,11 @@
 {
     public static final String HIDDEN_WIDGETS = "hidden_widget_types";
     public static final String NEW_DISPLAY_TEMPLATE = "new_display_template";
+    public static final String UNDO_STACK_SIZE = "undo_stack.size";
```

Comment:
So far all the preference settings follow the pattern `some.package.name/lower_case` with dots in the package name and underscores in the variable name. 

https://control-system-studio.readthedocs.io/en/latest/preferences.html
https://control-system-studio.readthedocs.io/en/latest/preference_properties.html.